samples/spring-eventsourced-counter/src/it/java/com/example/TestConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.example;
+// tag::class[]
+import kalix.javasdk.testkit.KalixTestKit;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class TestConfig {
+    @Bean
+    public KalixTestKit.Settings settings() {
+        return KalixTestKit.Settings.DEFAULT.withAclEnabled(); // <1>
+    }
+}
```

Comment:
I'm wondering which other name we can use here.

`TestConfig` is not bad because we are providing extra Spring `@Configuration` to the testkit. 

On the other hand, the only thing that we can provide are the `Settings`. So maybe, `TestkitSettings`. 

Or maybe, `TestkitConfiguration` or `TestkitConfig` to enforce that this `@Configuration` is for tweaking the testkit.

Reply:
Let's use `TestkitConfig`